### DISCUSSION TITLE
Features under consideration | Tell me which features you want me to add

### DISCUSSION THREAD

**Discussion by orenelbaum** [orenelbaum/babel-plugin-solid-undestructure#5] [General]
Features under consideration:
- Nested destructuring
- A pragma to annotate components
- Automatically apply the transformation to every function annotated with a type with a `Component` suffix
- Automatically apply the transformation to every function assigned to an identifier that starts with a capital letter

Please tell me if you want any feature to be added to this plugin, whether it's on the list or not.

**Comment by gandazgul** [comment]
I'm very interested in having an option to automatically apply the transformation to every function assigned to an identifier or named starting with a capital letter. I can submit a PR, I'll use patch-package.

**Comment by gandazgul** [comment]
@orenelbaum just submitted a PR for this sorry it took so long, life got in the way. https://github.com/orenelbaum/babel-plugin-solid-undestructure/pull/17
